src/models/transfers.js

```diff
@@ -29,6 +29,8 @@ const Condition = require('five-bells-condition').Condition
 
 const validExecutionStates = ['prepared']
 const validCancellationStates = ['prepared', 'proposed']
+const SIGNATURE_TYPE_ED25519 = 'ed25519-sha512'
+const SIGNATURE_TYPE_SHA256 = 'sha256'
```

Comment:
This is one of the papers we referenced in the whitepaper that shows "[One-Way Functions are Necessary and Sufficient for Secure Signatures](http://www.cs.princeton.edu/courses/archive/spr08/cos598D/Rompel.pdf)". Not sure what the best approach is for naming. We might need to just put some documentation somewhere about how hash algorithms can be used for signatures and put links to that anywhere we think people might get confused